scripts/simple-as-coding-database/simple-as-coding-db-docker-linux.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+CONTAINER_NAME="simpleascoding"
+
+# Check if the container already exists, if so, delete it
+if docker ps -a --format "{{.Names}}" | grep -q "^$CONTAINER_NAME$"; then
+    echo "Deleting old container: $CONTAINER_NAME"
+    docker stop $CONTAINER_NAME > /dev/null 2>&1
+    docker rm $CONTAINER_NAME > /dev/null 2>&1
+fi
+
+# Starting a new container
+docker run -d \
+--name $CONTAINER_NAME \
+-e POSTGRES_USER=postgres \
+-e POSTGRES_PASSWORD=root \
+-e POSTGRES_DB=simpleascoding \
+-p 5432:5432 \
+postgres
+echo "Created new container: $CONTAINER_NAME"
```

Comment:
What if the command:
`docker run ...` 
ends unsuccessfully?
Message:
`echo "Created new container: $CONTAINER_NAME"`
will be wrong!

Maybe use conditional mode to display the message?

### Links
- https://unix.stackexchange.com/questions/22726/how-to-conditionally-do-something-if-a-command-succeeded-or-failed
- https://ss64.com/nt/if.html

Reply:
True! I supplemented the script with displaying several types of errors.